collectd-no-processes.conf

```diff
@@ -0,0 +1,82 @@
+# This file is a variant of the default collectd.conf except processes metrics are not collected.
```

Comment:
My worry is that we would have to keep these two files in sync as we make changes, which is error-prone… Not much we can do about it, though.

Reply:
Yeah. Another way to think about this is that if we do not do it, a number of customers would need to do it. So...